### DISCUSSION TITLE
Extension information only via "reload extension info" visible

### DISCUSSION THREAD

**Discussion by flow-motion** [akeeba/panopticon#351] [Q&A (Community Support)] [answered]
Hi everyone - My Joomla Core overview works perfect, but at the extensions, i have to click on reload on every page i have to see if there is an update. Anyone an idea, if panopticon can´t refresh that or, may be, my installation has troubles?

**Comment by nikosdion** [comment, marked as answer]
There was a bug in 1.0.4. I plan on releasing 1.0.5 tomorrow with a fix for that. You will probably have to restart the extensions refresh task on each site once more if it's already stuck, then it should work fine.

**Reply by flow-motion** [reply]
U r a hero. Thanks man.
After the update i get strange headings an texts... things like : panopticon_mail_lbl_filter_coreupdates are in the dashboard now. Do i have to refresh the database, or things like that?

**Reply by flow-motion** [reply]
I see there is a thread regarding the update failure in german. 
I can offer me as a tester for future german lang. Updates. 
Thanks guys!

**Reply by nikosdion** [reply]
I am not sure what happened there. All tests I have made show that the fallback language (en-GB) is loaded fine. I will publish a new version really soon, with updated translation instructions, anyway.

**Reply by flow-motion** [reply]
Shit happens. Don´t worry .)
